### ISSUE TITLE
[css-font-loading] Renaming the FontFace 'stretch' attribute to 'width' (to align with CSS Fonts)

### ISSUE THREAD

**Issue by jfkthame** [w3c/csswg-drafts#10673]
With CSS Fonts having [replaced the `font-stretch` descriptor with `font-weight`](https://drafts.csswg.org/css-fonts/#font-prop-desc), we should presumably make a corresponding change to the [FontFace](https://drafts.csswg.org/css-font-loading/#fontface-interface) interface defined by CSS Font Loading.

**Comment by svgeesus** [comment]
Yes, indeed we should.

**Comment by svgeesus** [comment]
How do we deal with the legacy alias though, on that interface?

**Comment by jfkthame** [comment]
Presumably the [`FontFace` interface](https://drafts.csswg.org/css-font-loading/#fontface) can simply expose both `width` and `stretch` attributes, with both being mapped to the same internal value (so that setting either of them will affect the value returned by both).

For the [`FontFaceDescriptors` dictionary](https://drafts.csswg.org/css-font-loading/#dictdef-fontfacedescriptors) passed to the constructor, either `width` and `stretch` can be used to set the attribute. If both are present but have different values, what happens? We could either say this throws an error, or define which is to take precedence. (I think I'd vote for giving precedence to `width` as the "canonical" name of the attribute, and only honoring `stretch` if `width` is not present.)

**Comment by tabatkins** [comment]
Yeah, we can have them both map to the same internal slot, that's fine.

For the dictionary, I think I weakly lean towards "defer to `width` when both are present", to allow for transitional usage.

**Comment by svgeesus** [comment]
@jfkthame would you be able to make a PR for this change?

**Comment by svgeesus** [comment]
> I think I'd vote for giving precedence to `width` as the "canonical" name of the attribute, and only honoring `stretch` if `width` is not present

Like @tabatkins I also prefer this option.
